docs/getting-started/quick-start.md

```diff
@@ -3,28 +3,97 @@ category: installation
 order: 10
 menu-title: Quick Start
 meta-title: Quick Start | CKEditor 5 documentation
-meta-description: Learn how to install, integrate, configure, and develop CKEditor 5. Browse through the API documentation and online samples.
+meta-description: Learn the fastest way to install a powerful, rich text WYSIWYG editor - CKEditor 5 - in your web application using npm.
 ---
 
-# Getting started with CKEditor&nbsp;5
+# Quick Start
 
-CKEditor&nbsp;5 provides every type of WYSIWYG editing solution imaginable. From editors similar to Google Docs and Medium to Slack or Twitter-like applications, all is possible within a single editing framework. It is an ultra-modern JavaScript rich-text editor with MVC architecture, custom data model, and virtual DOM, written from scratch in TypeScript with excellent webpack and Vite support. Find out the most convenient way to start using it!
+CKEditor&nbsp;5 is a powerful, rich text editor you can embed in your web application. This guide will show you the fastest way to start using it.
 
-<span class="navigation-hint_mobile">
-	<info-box>
-		Use the **main menu button in the upper-left corner** to navigate through the documentation.
-	</info-box>
-</span>
+## Try CKEditor&nbsp;5 builder
 
-<!-- the wrapper class should be transformed into an additional infobox class if we will want to use it on other pages-->
-<!-- <div class="info-cta"> -->
-<info-box>
-	**Migrating from CKEditor 4?** {@link updating/migration-from-ckeditor-4 Check the migration section}
-</info-box>
+Check out our interactive builder to quickly get a taste of CKEditor&nbsp;5. It offers an easy-to-use user interface to help you configure, preview, and download the editor suited to your needs.
+
+## Installing CKEditor&nbsp;5 using npm
 
 <info-box>
-	**Want to remove the "Powered by CKEditor" logo?** {@link support/managing-ckeditor-logo Check the dedicated guide}
+
+**Prerequisites:**
+
+* [Node.js](https://nodejs.org/) in version 18 or above.
+* An initialized Node.js project.
+* Basic familiarity with a terminal.
+
 </info-box>
-<!-- </div> -->
 
-TODO
+In your initialized Node.js project, run the following command:
+
+```bash
+npm install ckeditor5
+```
+
+The command will install the main CKEditor&nbsp;5 package containing all open-source plugins.
+
+After that, you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately. **Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the example below.
+
+```js
+import { ClassicEditor, Essentials, Bold, Italic, Heading, Paragraph } from 'ckeditor5';
+import translations from 'ckeditor5/dist/translations/pl.js';
```

Comment:
I would comment out this line but keep, I think I wouldn't want people to see their first editor in Polish :D We could pick Spanish as a second largest for native speakers.